permission/db.go

```diff
@@ -77,7 +77,8 @@ func (s *Store) editPermission(ctx context.Context, p1 Permission) (Permission,
 
 func (s *Store) deletePermission(ctx context.Context, p Permission) error {
 	builder := sq.Delete("people.permissions").
-		Where(sq.Eq{"permission_id": p.PermissionID})
+		Where(sq.Eq{"permission_id": p.PermissionID}).
+		PlaceholderFormat(sq.Dollar)
```

Comment:
NIT: you could avoid needing to repeat the `PlaceholderFormat` by defining a package-global `var psql = sq.StatementBuilder.PlaceholderFormat(sq.Dollar)` and then doing `psql.Delete(...)`.

Reply:
Fixed